data/json/items/book/abstract.json

```diff
@@ -167,6 +167,24 @@
     "copy-from": "book_nonf_soft_tpl",
     "relative": { "price": 400, "price_postapoc": -2600, "intelligence": 3 }
   },
+  {
+    "abstract": "book_pulp_newage_tpl",
+    "type": "BOOK",
+    "name": { "str": "Generic New-Age Book" },
+    "description": "This is a template for pulp new-age books.  They're theoretically nonfiction.",
+    "copy-from": "book_fict_soft_pulp_tpl",
+    "weight": "250 g",
+    "relative": { "chapters": -6, "price": 2500, "price_postapoc": -100 }
+  },
+  {
+    "abstract": "book_fict_soft_pulp_tpl",
+    "type": "BOOK",
+    "name": { "str": "Generic Pulp Book" },
+    "description": "This is a template for pulp books.  Which really all ought to be paperbacks, right?",
+    "looks_like": "novel_pulp",
+    "copy-from": "book_fict_soft_tpl",
+    "relative": { "intelligence": -1, "chapters": -2, "fun": 1, "price": -200, "price_postapoc": -1000 }
```

Comment:
Are post apoc soft fictions priced more than $10?  Are there any consequences for a trader holding a book with negative worth or does it bottom at zero?

Reply:
Possibly use proportional instead of relative for the prices? Another question: Should the weight be reduced (again, possibly with proportional) for them being paperbacks?